test/playwright.spec.js

```diff
@@ -113,8 +113,11 @@ module.exports.describe = ({testRunner, product, playwrightPath}) => {
 
     beforeEach(async(state, test) => {
       const onLine = (line) => test.output += line + '\n';
-      if (dumpProtocolOnFailure)
+      if (dumpProtocolOnFailure) {
+        state.browserServer.process().stdout.on('data', onLine);
+        state.browserServer.process().stderr.on('data', onLine);
         state.browser._setDebugFunction(onLine);
+      }
 
       let rl;
       if (state.browserServer.process().stderr) {
```

Comment:
this already happens here. I guess you want stdout as well, you can add it there.